spec.bs

```diff
@@ -338,17 +338,17 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component<
 <div algorithm>
   The <dfn constructor for=URLPattern lt="URLPattern(input, baseURL, options)">new URLPattern(|input|, |baseURL|, |options|)</dfn> constructor steps are:
 
-  1. Run [=initialize=] given [=this=], |input|, |baseURL|, and |options|.
+  1. Run [=initialize=] given [=this=]'s associated [=URL Pattern=], |input|, |baseURL|, and |options|.
 </div>
 
 <div algorithm>
   The <dfn constructor for=URLPattern lt="URLPattern(input, options)">new URLPattern(|input|, |options|)</dfn> constructor steps are:
 
-  1. Run [=initialize=] given [=this=], |input|, null, and |options|.
+  1. Run [=initialize=] given [=this=]'s associated [=URL Pattern=], |input|, null, and |options|.
 </div>
 
 <div algorithm>
-  To <dfn for=URLPattern>initialize</dfn> a {{URLPattern}} given a {{URLPattern}} |this|, {{URLPatternInput}} |input|, string or null |baseURL|, and {{URLPatternOptions}} |options|:
+  To <dfn for=URLPattern>initialize</dfn> a [=URL Pattern=] given a [=URL Pattern=] |this|, {{URLPatternInput}} |input|, string or null |baseURL|, and {{URLPatternOptions}} |options|:
```

Comment:
Probably _initialize_ should initialize {{URLPattern}}. Can we create a new algorithm like `create` and call it internally in _initialize_?

Reply:
done.